curvefs/src/client/rpcclient/mds_client.cpp

```diff
@@ -94,7 +94,10 @@ FSStatusCode MdsClientImpl::CreateFsS3(const std::string &fsName,
 
         return ret;
     };
-    return ReturnError(rpcexcutor_.DoRPCTask(task, mdsOpt_.mdsMaxRetryMS));
+
+    // manually set to a large timeout, because create fs may very slow
```

Comment:
// TODO
createfs will remove in the feature.

Reply:
line 78 has a TODO about this